src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM microsoft/mssql-server-linux:latest
+
+RUN mkdir -p /usr/config
+WORKDIR /usr/config
+
+COPY /ShellScripts .
+COPY /Deployment sql/Deployment
+COPY /Indexes sql/Indexes
+COPY /Tables sql/Tables
+COPY /Security sql/Security
+
+RUN chmod +x /usr/config/entrypoint.sh
+RUN chmod +x /usr/config/configure-db.sh
```

Comment:
This isn't anything major, but we could improve this by combining them into 1 run statement (to save docker the trouble of making a whole new layer just for a 'chmod'). Entirely up to you to make any changes based on my suggestion. [Some reading on this topic.](https://stackoverflow.com/questions/39223249/multiple-run-vs-single-chained-run-in-dockerfile-what-is-better)
 
So instead of 
```
RUN chmod +x /usr/config/entrypoint.sh
RUN chmod +x /usr/config/configure-db.sh
```
Combine them
```
RUN chmod +x /usr/config/entrypoint.sh && chmod +x /usr/config/configure-db.sh
```